### PR TITLE
Small changes

### LINKED ISSUE #67: HttpRequestUrlEnricher gives "Request is not available in this context" exception
I am currently using SerilogWeb.Classic v5.0.56 in a legacy webforms website.
After I decided to enable SelfLog, I found that adding log entries from the `Application_Start` in `Global.asax` while enriching with `HttpRequestUrl` chained to `new LoggerConfiguration()` throws the following exception
```C#
2020-04-17T16:29:46.1743072Z Exception System.Web.HttpException (0x80004005): Request is not available in this context
   at System.Web.HttpContext.get_Request()
   at SerilogWeb.Classic.Enrichers.HttpRequestUrlEnricher.Enrich(LogEvent logEvent, ILogEventPropertyFactory propertyFactory)
   at Serilog.Core.Enrichers.SafeAggregateEnricher.Enrich(LogEvent logEvent, ILogEventPropertyFactory propertyFactory) caught while enriching Serilog.Events.LogEvent with SerilogWeb.Classic.Enrichers.HttpRequestUrlEnricher.
```

'This well known exception raises its fancy head again', I told myself. So, I've looked into `HttpContextCurrent.cs` and `HttpRequestUrlEnricher.cs`.

I think that replacing `if (HttpContext.Current?.Request?.Url == null)` with `if (HttpContextCurrent.Request?.Url == null)` would fix it.

_I am new to github. If I make mistakes please forgive a newbie, I'm learning/discovering :)_

**PS**: Also I looked through the other files in the project and found no other instance of `Request` being accessed outside project's `HttpContextCurrent`. I was thinking of renaming this to help against any future confusion with `HttpContext.Current`.

### PR DESCRIPTION
**Pull request by liviriniu**
If my changes are not well formed, I'm all open to learning about best practices. I've separated interventions into separate commits.

fixes #67

### CODE REVIEW

**Comment by liviriniu** [comment]
I'll return after ~9h with an issue. Maybe integrate additional changes in this PR?

**Comment by tsimbalar** [comment]
@liviriniu I'd rather have it in a separate PR if you don't mind, that's a bit eaiser to review :)

File: src/SerilogWeb.Classic/Classic/Enrichers/CurrentHttpContext.cs [hunk 1]
```diff
@@ -23,12 +23,9 @@ internal static HttpRequest Request
         {
             get
             {
-                HttpContext httpContext = HttpContext.Current;
-                if (httpContext == null)
-                    return null;
                 try
                 {
-                    return httpContext.Request;
```
Comment (tsimbalar) [inline src/SerilogWeb.Classic/Classic/Enrichers/CurrentHttpContext.cs:31]: Nice improvement 👍

File: test/SerilogWeb.Classic.Tests/ModuleConfigurationContractTests.cs [hunk 2]
```diff
@@ -396,7 +396,7 @@ public void RequestFiltering()
         [InlineData(500, true)]
         [InlineData(501, true)]
         [InlineData(499, false)]
-        public void StatusCodeBiggerThan500AreLoggedAsError(int httpStatusCode, bool isLoggedAsError)
+        public void StatusCodeEqualOrBiggerThan500AreLoggedAsError(int httpStatusCode, bool isLoggedAsError)
```
Comment (tsimbalar) [inline test/SerilogWeb.Classic.Tests/ModuleConfigurationContractTests.cs:399]: 👍

**Review by tsimbalar — APPROVED** [review]
Nice improvements, thanks ! 

I really wish we could write tests for those enrichers to prove that they won't explode when enriching in a situation where `HttpContext.Current.Request` is unavailable, but `HttpContext` is really not test-friendly, with basically no option to fake it or anything so 🤷  


Let's Get That Merged ! 🚀 


(also, nice touch in providing it as a series of small _atomic_ commits 💪  )

**Pull request merged**